bbc/simorgh
# Chartbeat integration

Resolves #1987

**Overall change:** Integrated Chartbeat Analytics

**Code changes:**
  - added charbeat beacon to head tag
  - created beacon and initialization code for amp and canonical pages

**To test**
  - You can check that requests are being sent to Chartbeat by observing your network requests panel and looking for the domain `ping.chartbeat.com`.
  - Install [Chartbeat Heads-Up Display](https://chartbeat.zendesk.com/hc/en-us/articles/115001293233-Install-the-Heads-Up-Display-HUD-) and observe that it fires on the page

---
